go.mod

```diff
@@ -24,13 +24,13 @@ require (
 	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83
 	golang.org/x/sys v0.0.0-20201112073958-5cba982894dd
 	google.golang.org/grpc v1.36.1
-	k8s.io/api v0.20.0
-	k8s.io/apimachinery v0.20.0
-	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/cloud-provider v0.20.0
+	k8s.io/api v0.20.6
+	k8s.io/apimachinery v0.20.6
+	k8s.io/client-go v0.20.6
+	k8s.io/cloud-provider v0.20.6
 	k8s.io/klog/v2 v2.4.0
 	k8s.io/kubernetes v1.20.0
```

Comment:
can you also update `k8s.io/kubernetes` to v1.20.6?

Reply:
Done. thanks .